disnake/client.py

```diff
@@ -525,6 +479,28 @@ def _handle_first_connect(self) -> None:
             return
         self._first_connect.set()
 
+    @property
+    def loop(self):
+        """:class:`asyncio.AbstractEventLoop`: Same as :func:`asyncio.get_running_loop`.
+
+        .. deprecated:: 3.0
+            Use :func:`asyncio.get_running_loop` directly.
+        """
+        warnings.warn(
+            "Accessing `Client.loop` is deprecated. Use `asyncio.get_running_loop()` instead.",
+            category=DeprecationWarning,
+            stacklevel=2,
+        )
+        return asyncio.get_running_loop()
+
+    @loop.setter
+    def loop(self, _value: Never) -> None:
+        warnings.warn(
+            "Setting `Client.loop` is deprecated and has no effect. Use `asyncio.get_running_loop()` instead.",
```

Comment:
TODO: i'm not sure the suggestion is correct (apart from the typo (`get` => `set`), i don't know if `set_event_loop` will work on-the-fly); might be better to remove the suggestion altogether